profile/modules/vsite/modules/vsite_preset/tests/src/ExistingSite/VsitePresetHelperTest.php

```
@@ -0,0 +1,175 @@
+<?php
+
+namespace Drupal\Tests\vsite_preset\ExistingSite;
+
+use Drupal\Tests\vsite\ExistingSite\VsiteExistingSiteTestBase;
+use Drupal\vsite_preset\Entity\GroupPreset;
+
+/**
+ * VsitePresetHelperTest.
+ *
+ * @group vsite
+ * @group kernel
+ */
+class VsitePresetHelperTest extends VsiteExistingSiteTestBase {
+
+  /**
+   * Apps to enable.
+   *
+   * @var array
+   */
+  protected $toEnable;
+
+  /**
+   * File uri array.
+   *
+   * @var array
+   */
+  protected $uriArr;
+
+  /**
+   * Vsite helper service.
+   *
+   * @var \Drupal\vsite_preset\Helper\VsitePresetHelper
+   */
+  protected $vsitePresetHelper;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+    $this->vsitePresetHelper = $this->container->get('vsite_preset.preset_helper');
+    $this->group = $this->createGroup();
+    $this->vsiteContextManager->activateVsite($this->group);
+    $this->toEnable = [
+      'class' => 'class',
+      'publications' => 'publications',
+    ];
+
+    /** @var \Drupal\vsite_preset\Entity\GroupPreset $preset */
+    $preset = GroupPreset::load('personal');
+    $paths = $preset->getCreationFilePaths();
```

I think you should a test `csv`s here.

---

Since this is different from Import, where each time uploaded files are different no files exist in the system to test properly, is it really necessary to create a new test module and stuff for it , when we are specifically testing `personal` group here, why not use it's current files only.

---

actually this came into picture after I saw this code https://github.com/openscholar/openscholar/pull/12794/files#r367785392

It looks like the test is suppressing the error for a certain type. It is however known to us, but will look suspicious to a new developer.

I think you can combine `testVsitePresetDefaultContentCreation` and `testVsitePresetDefaultWidgetCreation` and do the assertion in one test case.